src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -116,35 +116,37 @@ export const pageContent = {
       'I understand that if I act contrary to the specification, I am liable to be slashed.',
   },
   [AcknowledgementIdsEnum.keyManagement]: {
-    title: 'Keys safety',
+    title: 'Keys management',
     content: (
-      <Text size="large" className="my10">
-        Your validator keys are derived from a unique mnemonic (or seed). Your
-        seed will be the ONLY WAY to withdraw your funds. Above all, keep it
-        safe!
-      </Text>
-    ),
-    acknowledgementText:
-      'I understand that my mnemonic (seed) will be the ‼️ONLY WAY‼️ to withdraw my funds.',
-  },
-  [AcknowledgementIdsEnum.signingKeys]: {
-    title: 'Signing keys',
-    content: (
-      <Text size="large" className="my10">
-        We'll help you create keys for each of your validators. These keys will
-        be saved in key-stores and you will need to give these to your validator
-        software to begin validating. You will also receive a deposit file to
-        upload to this website with the public keys for your validator.
-      </Text>
+      <>
+        <Text size="medium" className="mt10">
+          To become a validator you'll need to know about managing keys. If
+          you're unfamiliar with keys and mnemomics, please don't proceed. Your
```

Comment:
I'm worried that users read this far, think "I know how mnemonics work" and stop reading further. I think we should be emphasising that this is slightly different to that in that the validator keys do now allow you to withdraw, only the mnemonic does that.